src/igvfd/schemas/whole_organism.json

```diff
@@ -0,0 +1,68 @@
+{
+    "title": "Whole organism",
+    "$id": "/profiles/whole_organism.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submiting a whole organism sample.",
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "source"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "accession",
+        "alternate_accessions",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "sample.json#/properties"
+        },
+        {
+            "$ref": "biosample.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        },
+        {
+            "$ref": "mixins.json#/url"
+        },
+        {
+            "$ref": "mixins.json#/references"
+        },
+        {
+            "$ref": "mixins.json#/taxa"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "biosample.json#/dependentSchemas"
```

Comment:
I think we need to refine dependency not to include anything human

Reply:
and add a dependency that makes sure one can not submit human taxa whole organism

Reply:
Done